i18n/fr.jsonp.js

```diff
@@ -178,6 +178,11 @@ document.localeJson = {
       "dialog-readzim-error-message": "Erreur lors de la lecture du fichier ZIM",
       "dialog-reset-warning-title": "ATTENTION&nbsp;!",
       "dialog-reset-warning-message": "Cela réinitialisera l'application à l'état d'installation originale. Tous les caches et paramètres seront effacés&nbsp;!",
+      "dialog-metadata-warning": "Attention: les données ci-dessus peuvent être falsifiées!",
+      "dialog-metadata-name": "Nom: ",
+      "dialog-metadata-creator": "Créateur: ",
+      "dialog-metadata-publisher": "Éditeur: ",
+      "dialog-metadata-scraper": "Grattoir: ",
```

Comment:
In all the above translations, please put `&nbsp;` between the word and the colon (but the space after the colon can remain as a normal space). Also, in the first case, add `&nbsp;` before the exclamation mark. This is because French uses spacing around colons, question marks, exclamation marks and guillemets. It needs to be a non-breaking space so that the punctuation doesn't get orphaned on narrow screens.